sdk/nodejs/test/connect.spec.ts

```diff
@@ -7,11 +7,56 @@ import * as os from "os"
 import * as path from "path"
 import * as tar from "tar"
 
+import * as dagger from "../api/client.gen.js"
 import { GraphQLRequestError } from "../common/errors/index.js"
-import { connect } from "../connect.js"
+import { connect, close, connection } from "../connect.js"
 import * as bin from "../provisioning/bin.js"
 import { CLI_VERSION } from "../provisioning/default.js"
 
+describe("NodeJS default client", function () {
+  it("Should use the default client", async function () {
+    this.timeout(60000)
+
+    await dagger
+      .container()
+      .from("alpine")
+      .withExec(["apk", "add", "curl"])
+      .withExec(["curl", "https://dagger.io/"])
+      .sync()
+
+    close()
+  })
+
+  it("Should automatically close connection", async function () {
+    this.timeout(60000)
+
+    await connection(async () => {
+      await dagger
+        .container()
+        .from("alpine")
+        .withExec(["apk", "add", "curl"])
+        .withExec(["curl", "https://dagger.io/"])
+        .sync()
```

Comment:
I know the tests already in this file use the same pipeline, but:
- Perhaps best pinning the alpine version (caching)
- Could use a simple `echo` that's faster than `curl`ling a URL
- Testing that it doesn't fail doesn't account for silently not doing anything. Better to assert for something to make sure it did work.

Reply:
Done in: [9b750b9](https://github.com/dagger/dagger/pull/6118/commits/9b750b937b9f175f73569e71465c5467b81c58c1)